roxFile/createroxfile-service.ts

```diff
@@ -1,12 +1,19 @@
 import * as PH from "processhub-sdk";
 import * as Types from "./roxtrafileapitypes";
-import { readFileBase64Async, missingRequiredField, initRequiredFields, RoXtraFileApi, errorHandling } from "./roxtrafileapi";
+import { missingRequiredField, initRequiredFields, RoXtraFileApi, errorHandling } from "./roxtrafileapi";
 import { IRoXtraFileApi } from "./iroxtrafileapi";
+import { getBackendUrl } from "processhub-sdk/lib/config";
 
 export let errorState: number = Types.ERRORCODES.NOERROR;
 let APIUrl: string;
 let efAccessToken: string;
 
+async function getAttachedFile(environment: PH.ServiceTask.IServiceTaskEnvironment, relativePath: string): Promise<Buffer> {
+  const pathParts = relativePath.split("/");
+  const filePath = decodeURIComponent(relativePath.replace(pathParts[0] + "/", "").replace(pathParts[1] + "/", ""));
+  return await environment.fileStore.getFileBuffer(pathParts[0], pathParts[1], filePath);
+}
+
```

Comment:
Ok, I think this is exactly the same code as in `fieldtofile-service.ts`. Wouldn't it therefore make sense to move this to the SDK if possible or just download the attachment via the given Url?